h2-migration-tool/migration.sh

```diff
@@ -17,6 +17,55 @@
 echo
 echo "Welcome to the H2 DB Migration Tool!"
 echo
+echo "Please select the h2 version you are migrating from:"
+echo "h2-2.1.210 - (1)"
+echo "h2-1.4.199 - (2)"
+echo "h2-1.3.175 - (3)"
+read old_h2_version_choice
+
+case $old_h2_version_choice in
+    1)
+        old_h2_version="h2-2.1.210.jar"
+        wget -c https://repo1.maven.org/maven2/com/h2database/h2/2.1.210/h2-2.1.210.jar || exit 1
+        ;;
+    2)
+        old_h2_version="h2-1.4.199.jar"
+        wget -c https://repo1.maven.org/maven2/com/h2database/h2/1.4.199/h2-1.4.199.jar || exit 1
+        ;;
+    3)
+        old_h2_version="h2-1.3.175.jar"
+        wget -c https://repo1.maven.org/maven2/com/h2database/h2/1.3.175/h2-1.3.175.jar || exit 1
+        ;;
+    *)
+        echo "Invalid choice. Exiting."
+        exit 1
+        ;;
+esac
+
+echo
+echo "Please select the h2 version you are migrating to:"
+echo "h2-2.2.220 - (1)"
+echo "h2-2.1.210 - (2)"
+read new_h2_version_choice
+
+case $new_h2_version_choice in
+    1)
+        new_h2_version="h2-2.2.220.jar"
+        wget -c https://repo1.maven.org/maven2/com/h2database/h2/2.2.220/h2-2.2.220.jar || exit 1
+        ;;
+    2)
+        new_h2_version="h2-2.1.210.jar"
+        wget -c https://repo1.maven.org/maven2/com/h2database/h2/2.1.210/h2-2.1.210.jar || exit 1
+        ;;
+    *)
+        echo "Invalid choice. Exiting."
+        exit 1
+        ;;
+esac
+
+# Always download the latest h2 version we're migrating to
+wget -c https://repo1.maven.org/maven2/com/h2database/h2/2.2.220/h2-2.2.220.jar || exit 1
```

Comment:
Do we need to download this always?

Reply:
no. this is already getting downloaded with the improvement added with https://github.com/wso2/samples-is/pull/419/commits/6a26399b4780ea72fff25f8cc9fdeb592f038b50.
Fixed with https://github.com/wso2/samples-is/pull/419/commits/c579cb0bf7a28a43957093f6c82f6e6bacd128bf